galley/pkg/config/analysis/analyzers/testdata/virtualservice_dupmatches.yaml

```diff
@@ -0,0 +1,218 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: sample-foo-cluster01
+  namespace: foo
+spec:
+  hosts:
+  - sample.foo.svc.cluster.local
+  http:
```

Comment:
a route can have a name. Seems like we ought to use that instead of index if its set

Reply:
@howardjohn Good catch.  I revised it.  Messages now look like:

Reply:
fixed